moped-editor/src/views/projects/projectsListView/ProjectsListViewTable.js

```diff
@@ -534,8 +534,37 @@ const ProjectsListViewTable = ({ query, searchTerm }) => {
       title: "Interim MPD (Access) ID",
       field: "interim_project_id",
       hidden: hiddenColumns["interim_project_id"],
-      emptyValue: "-"
-    }
+      emptyValue: "-",
+    },
+    {
+      title: "Parent project",
+      field: "parent_project_id",
+      hidden: hiddenColumns["parent_project_id"],
+      emptyValue: "-",
+      render: (entry) => (
+        <RouterLink
+          to={`/moped/projects/${entry.parent_project_id}`}
+          state={{
+            filters: Object.keys(filters).length
+              ? btoa(JSON.stringify(filters))
```

Comment:
TIL [`btoa`](https://developer.mozilla.org/en-US/docs/Web/API/btoa)!

it looks like every cell in the table is using the same value for this link state. if you assigned this value outside the `columns` declaration you might see a speedup with rendering.

```jsx
# earlier
const linkStateFilters =  Object.keys(filters).length ? btoa(JSON.stringify(filters)) : false;


# inside the columns render definition
<RouterLink
          to={`/moped/projects/${entry.parent_project_id}`}
          state={{ filters: linkStateFilters }}>
```